processed 1 task

task 0 'publish'. lines 4-31:
Error: Error executing ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: ModuleVerificationFailure, source: None } }
